R/grid_code_paths.R

```diff
@@ -363,6 +365,10 @@ tune_grid_loop_iter_safely <- function(split,
     control,
     seed
   )
+  new.time <- proc.time()
+
+  # Update with elapsed time
+  result$result[[".elapsed"]] <- new.time - time
```

Comment:
It would be simpler to just use
```suggestion
  result$result[[".elapsed"]] <- new.time["elapsed"] - time["elapsed"]
```